src/cmp_aus.js

```diff
@@ -1,6 +1,19 @@
 const synthetics = require('Synthetics');
 const log = require('SyntheticsLogger');
 
+const checkCMPIsHidden = async (page) => {
+	const display = await page.evaluate(
+		`window.getComputedStyle(document.querySelector('[id*=\\"sp_message_container\\"]')).getPropertyValue('display')`,
+	);
```

Comment:
That’s a curious way to find elements on the page… Maybe something which doesn’t use a string would be easier to read?

https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pageevaluatepagefunction-args

```ts
	const display = await page.evaluate(
		(selector) => {		 		 
			window.getComputedStyle(document.querySelector(selector))
			.getPropertyValue('display')
		}, '[id*=\\"sp_message_container\\"]'
	);
```

Reply:
This is what I tried first of all but it gives an error. It seems as though the canary cannot properly access `window` or `document` within the scope of this function, so I had to resort to passing it as a string where it does seem to work okay

Reply:
🤷‍♂️